### ISSUE TITLE
Smalls is just working on markdown

### ISSUE THREAD

**Issue by liam-ngy** [t9md/atom-smalls#1]
Hey, smalls is just working for plain text, but if I want to use smalls on ruby files, it doesn't work.
It shows me selected words but after I press confirm it doesn't do anything.

**Comment by luxueyan** [comment]
@t9md I have the same problem. Only in markdown the label can show,but in other file the label can't show

**Comment by t9md** [comment]
Can't you invoke via command-pallette?
Not sure its keymap issue or other issue.
I can use smalls normal editor like some.coffee or some.rb.

**Comment by luxueyan** [comment]
@t9md It's not keymap issue, I debugged and found that in the lable.coffee, you append labels directly to the 'content' tag in the shadow dom. I change the overlayer to the real overlayer! The label show, but it's position not right

``` coffee
@overlayer = editorView.querySelector('.overlayer')
    # @overlayer = editorView.shadowRoot.querySelector('content[select=".overlayer"]')
```

**Comment by t9md** [comment]
Ok this is a bit old code. It seem I need to update to fit current Atom core. Thanks for giving me a hint.

One question.
Do you enabled shadowDOM?

**Comment by luxueyan** [comment]
No. Maybe the latest version  default to use  the shadowDOM. I begin to use Atom recently

**Comment by luxueyan** [comment]
Sorry! I find this 
![image](https://cloud.githubusercontent.com/assets/1555552/14490826/bd9ed54c-01a7-11e6-80d4-b6e1d7fad0e5.png)
if disabled this, there is a error
![image](https://cloud.githubusercontent.com/assets/1555552/14490860/f043877c-01a7-11e6-8059-b2f0e0e437f9.png)

**Comment by luxueyan** [comment]
``` coffee
#@file label.coffee --78
# I change @overlayer to the '.scroll-view' which is a real dom diffrent with the 'content', it works finally!
@overlayer = editorView.shadowRoot.querySelector('.scroll-view')
```

**Comment by t9md** [comment]
Although I'm currently not using this package, I decided to rewrite everything to learn/re-evaluate again effectiveness of this label jump method.

So please wait to overhaul done.

**Comment by luxueyan** [comment]
Thank you very much! This package is wonderful!

**Comment by t9md** [comment]
released v0.2.0, from this version, smalls use native overlay marker of Atom.
If your issue is disappear, close this issue please.
